measure.rb

```diff
@@ -1649,41 +1649,42 @@ def self.add_foundation_wall(runner, model, spaces, fnd_wall_values, slab_frac,
     else
       drywall_thick_in = 0.0
     end
```

Comment:
@shorowit Just want to confirm, we still want this drywall assumption (excluded from interior/exterior insulation) based on thermal boundary, right?

Reply:
Yes, and we should use it as part of the surface's construction (not the Kiva insulation blocks).